src/Data/String/Extra.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
```

Comment:
I'm OK with these functions being implemented in the FFI for performance reasons. However, if you believe it's possible to convert the implementation to PureScript without severe performance penalties that's always the preferred default. Given the choice between not having these at all and having the FFI implementations we can use the FFI here.

Reply:
I don't believe that's going to benefit performance. The API to purescript is pretty clean at the moment with a pure function interface.